helix-term/src/ui/editor.rs

```diff
@@ -830,6 +833,51 @@ impl EditorView {
         }
     }
 
+    /// Apply the highlighting on the columns where a cursor is active
+    pub fn highlight_cursorcolumn(
+        doc: &Document,
+        view: &View,
+        surface: &mut Surface,
+        theme: &Theme,
+    ) {
+        let text = doc.text().slice(..);
+
+        let primary_style = theme
+            .try_get("ui.cursorcolumn.primary")
+            .or_else(|| theme.try_get("ui.cursorcolumn"))
+            .unwrap_or_else(|| theme.get("ui.cursorline.primary"));
+        let secondary_style = theme
+            .try_get("ui.cursorcolumn.secondary")
+            .or_else(|| theme.try_get("ui.cursorcolumn"))
+            .unwrap_or_else(|| theme.get("ui.cursorline.secondary"));
```

Comment:
`try_get` actually does check the fallback scopes, the only difference to `get` is that `get` automatically unwraps into the default style if no style is found for any scope:

https://github.com/helix-editor/helix/blob/4d4be0e880efd136100439cc34e8c526d299ec34/helix-view/src/theme.rs#L268-L278

Reply:
Good catch! Fixed now